protocol-designer/src/pages/Designer/ProtocolSteps/Timeline/SubStepsToolbox.tsx

```diff
@@ -61,7 +65,8 @@ export function SubStepsToolbox(
       substeps.commandCreatorFnName === 'mix')) ||
     substeps.substepType === THERMOCYCLER_PROFILE ? (
     <Toolbox
-      height="calc(100vh - 6rem)"
+      maxHeight={`calc(100vh - ${NAV_BAR_HEIGHT_REM}rem - 2 * ${SPACING.spacing12})`}
```

Comment:
for this case, i think we can do 
```suggestion
      maxHeight={`calc(100vh - ${NAV_BAR_HEIGHT_REM}rem - 1.5rem)`}
```